packages/oidc-client/README.md

```diff
@@ -1,3 +1,26 @@
 # oidc-client
 
 A generic OpenID Connect (OIDC) client library for JavaScript and TypeScript, designed to work with any OIDC-compliant identity provider.
+
+```js
+// Initialize OIDC Client
+const oidcClient = oidc({
+  /* config */
+});
+
+// Authorize API
+const authResponse = oidcClient.authorize.background(); // Returns code and state if successful, error and Auth URL if not
+const authUrl = oidcClient.authorize.url(); // Returns Auth URL or error
+
+// Tokens API
```

Comment:
Do the tokens api and user api exist on the OIDC client yet? Didn't see them on the client store

Reply:
Yeah, this is just the `authorize` feature. Token exchange is found here: https://github.com/ForgeRock/ping-javascript-sdk/pull/348. The other token related methods are not implemeneted yet.